### ISSUE TITLE
id групп не понимает?

### ISSUE THREAD

**Issue by whoizit** [vrtx64/vk_exaile#8]
К примеру есть чудесная группа http://vkontakte.ru/audio?album_id=8516191&id=-912878

Как я понял API позволяет id груп, не?

**Comment by vrtx64** [comment]
Да, посмотрел сейчас, позволяет. Может раньше не позволял... Ну добавить не сложно вроде бы, возьму на дороботку

**Comment by whoizit** [comment]
Для меня не хватает списка музыки по id-группе. И выпадающий список с названиями/алиасами к id. Могу это сделать

**Comment by vrtx64** [comment]
Ну было бы здорово! а то пока доберусь.. :) Спасибо!

**Comment by whoizit** [comment]
и не сортируется в историческом порядке
